update download overview to vbt

pmahnke opened the pull request:
## Done

* converted /download to VBT

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- see the page http://0.0.0.0:8001/desktop/developers
- [see the demo](http://www.ubuntu.com-1620-desktop-overview-vbt.demo.haus/download)

## Issue / Card

Fixes #1620

barrymcgee commented on templates/download/index.html:
```diff
@@ -1,104 +1,111 @@
-                    <a href="/download/cloud"><img src="{{ ASSET_SERVER_URL }}dfd42685-picto-cloud-darkaubergine.svg" alt="" width="113" height="113" /></a>
-                </div>
-            </div>
+<div class="p-strip">
```
This strip needs `is-deep` according to @yaili's pattern spec - https://www.dropbox.com/home/00_web_team/_ubuntu.com/new%20projects/vanilla%20update/Patterns%20inventory/9.%20Download?preview=download-overview.png

pmahnke replied:
done

barrymcgee commented on static/sass/_v1_pattern_card.scss:
```diff
@@ -0,0 +1,15 @@
+@mixin ubuntu-p-card {
+  .p-card__footer--upgrade {
+    background: $color-light url('#{$assets-path}f68488b1-picto-upgrade-warmgrey.svg') 20px center no-repeat;
+    background-size: 40px 40px;
```
We should avoid using pixel values in CSS. Ideally, you should use [Vanilla spacing variables](https://github.com/vanilla-framework/vanilla-framework/blob/develop/scss/_settings_spacing.scss) - in this case `padding: $sp-xx-large $sp-xx-large`.

Where there is not an exact 1:1 match, choose the nearest value. If that value breaks the design, you may then be justified in using an alternate `rem` value but this is the last resort option.

pmahnke replied:
tried as much as I could

barrymcgee commented on static/sass/_v1_pattern_card.scss:
```diff
@@ -0,0 +1,15 @@
+    margin: 20px -20px 0 -20px;
+  }
+
+  html.no-svg,
```
These selectors are over-qualified, please remove `html`

pmahnke replied:
done

pmahnke commented:
@barrymcgee updated

yaili commented:
Some comments:
- [ ] The contextual footer is missing external icons
- [ ] Can the Buy Ubuntu Advantage link in the bottom be a neutral button style?

pmahnke commented:
@yaili I fixed the button; however the external icons are fixed in another PR... the demo has been updated

yaili commented:
- [ ] I just noticed that the light grey strip inside the Ubuntu Desktop card is not touching the edges, it's about 1px in L screen and it becomes a bit bigger gap as the screen size reduces

pmahnke commented:
@yaili I fixed it... and I also added some margin on the text bit as without, it seemed too tight?  obviously I can remove

my version
![image](https://cloud.githubusercontent.com/assets/441217/26370742/a3c3aab6-3ff0-11e7-9978-2b36af7ad27a.png)

no margin version
![image](https://cloud.githubusercontent.com/assets/441217/26370779/c52aadee-3ff0-11e7-8264-6658cc2c3c76.png)

yaili commented:
The demo link says "no demo here".

barrymcgee commented:
@yaili Just kicked it - http://www.ubuntu.com-1620-desktop-overview-vbt.demo.haus/
